h/static/scripts/annotation-metadata.js

```diff
@@ -15,25 +15,21 @@
 function extractDocumentMetadata(annotation) {
   var document_;
   var uri = annotation.uri;
-  var domain = new URL(uri).hostname;
-  if (annotation.document) {
-    if (uri.indexOf('urn') === 0) {
-      var i;
-      for (i = 0; i < annotation.document.link.length; i++) {
-        var link = annotation.document.link[i];
-        if (link.href.indexOf('urn:') === 0) {
-          continue;
-        }
-        uri = link.href;
-        break;
-      }
+
+  if (Array.isArray(annotation.target) && annotation.target.length > 0) {
+    var locator = annotation.target[0].locator;
+    if (locator) {
+      uri = locator;
     }
+  }
 
+  var domain = new URL(uri).hostname;
+
+  if (annotation.document) {
     var documentTitle;
-    if (Array.isArray(annotation.document.title)) {
+    if (Array.isArray(annotation.document.title) &&
```

Comment:
It is always an array now, like the other metadata (well, the one that I'm removing in this PR..). I'll change the check to be `annotation.document.title && ...` (since an empty array is falsy in JS, right?)